javascript/builtins/Intl/RelativeTimeFormat.json

```diff
@@ -369,44 +369,6 @@
                 "standard_track": true,
                 "deprecated": false
               }
-            },
-            "numberingSystem": {
```

Comment:
I removed this because we never document each individual `resolvedOptions` property. Instead, we document the `options` parameter of the constructor, and assume that `resolvedOptions` mirrors that. The data here doesn't make sense anyway—the Chrome version is later than `options.numberingSystem` but the Firefox version is _earlier_, so there must be something going on.